gb-ppu/src/registers/palette.rs

```diff
@@ -26,6 +26,13 @@ impl Palette {
         }
     }
 
+    pub fn from(byte: u8, is_sprite: bool) -> Self {
```

Comment:
`from` may conflict with `From::from`, maybe renaming this constructor to `new` and creating two other constructor `new_sprite` & `new_background` ?